source/quick-start.txt

```diff
@@ -7,193 +7,182 @@ Quick Start
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
 
-This guide shows you how to create an application that uses the Node.js
-driver to connect to a MongoDB Atlas cluster.
+*Estimated time to complete this guide: 35 minutes.*
 
-Check for Node and NPM
-----------------------
+This guide shows you how to create an application that uses the
```

Comment:
Would you consider the code we provide to users below an "application"? It may be an application in the technical sense, but may not match the mental models our users associate with that word. This could lead to an unsatisfactory outcome for users of this guide.

Reply:
Yes, it is an application, but what other words would you suggest based on users' mental models? If a Jasmine persona looks for specific keywords, I'd change this, but otherwise Lucas personas should understand it since it is accompanied with a description of its functionality.

Reply:
I was initially concerned given its usage so early on the page, but as I continued through the content, it looks like we use it consistently so I'm less concerned that there will be confusion.